workflow/scripts/pass_time_cylc.py

```diff
@@ -41,20 +41,18 @@ def __init___(self, args):
 
 
 def get_passtimes(
-    startdate, enddate, csvoutpath, centroid_x, centroid_y, SPACEUSER, SPACEPSWD
+    startdate, enddate, csvoutpath, centroid_lat, centroid_lon, SPACEUSER, SPACEPSWD
 ):
-    centroidx = centroid_x
-    centroidy = centroid_y
```

Comment:
Is there a reason for changing variable names `centroid_x` to `centroidx` to `lat` in the original script? Why not use `lat` all the way through? For now I switched `centroid_x` to `centroid_lat` to make it clear that providing an `x` value would break things.